wazo_bus/resources/common/abstract.py

```diff
@@ -49,6 +52,10 @@ def routing_key_fmt(self) -> str:
     def routing_key(self) -> str:
         variables = dict(**self.content)
         variables.update(vars(self), name=self.name)
+        variables = {
```

Comment:
Good idea to escape from the method instead of per event basis, will probably fix some other missed cases